bin/cli.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+
+"use strict";
+
+const { readFileSync, writeFileSync } = require("node:fs");
+
+const { sort } = require("..");
+
+const files = process.argv.slice(2);
+if (!files.length) {
+  console.error("Usage: better-sort-github-actions <file> [<file>...]");
+  process.exit(1);
+}
+
+// TODO check how globs are handled
```

Comment:
do we need an issue for this?

Reply:
It works. I just forgot to remove the comment. 🤦🏼‍♂️